app/javascript/crayons/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,35 @@
+import { h, cloneElement } from 'preact';
+import classNames from 'classnames/bind';
+
+export const Tabs = (props) => {
+  const {
+    children,
+    stacked,
+    fitted,
+    className,
+    elements = 'buttons',
+    ...otherProps
+  } = props;
+
+  const Wrapper = elements === 'buttons' ? 'div' : 'nav';
```

Comment:
I think we should actually _always_ wrap in a `<nav>`, because this component is always used for navigational purposes (even if we're staying on the same page, and using a button), and it's just an effective way to associate the buttons together.

Reply:
good to know, thanks!